_layouts/post.html

```diff
@@ -1,58 +1,66 @@
 ---
 layout: default
+slug: guides
 custom_js:
-- prism
+  - prism
+  - guides
 ---
 
-<div>
-  <div class="container-fluid section-ghostwhite">
-    <div class="col-md-3">
-      <div id="all-guides">
-        <label class="pull-right" >
-          <a href="/guides">&lt; All Guides</a>
-        </label>
+<div  class="container-fluid post-bg">
+  <div class="post-detail">
+    <div class="col-sm-2-5">
+      <div id="toc">
```

Comment:
There are some weird behaviours with this toc. 

1. Some items are not appearing aligned in the list
2. The marker doesnt correpond to the one on design when highlighted (should be a bullet point instead of an arrow)
3. The behavior of expanding on hover is seems really undesirable. I personally found it annoying to use. Is this really the intention of the design? Could we ask Sam about this one, please? If you did already, what did she say?

<img width="270" alt="Screen Shot 2019-09-05 at 14 10 35" src="https://user-images.githubusercontent.com/1516418/64355431-210fca00-cff9-11e9-8b20-82062b28514b.png">

Reply:
I never asked her about this but since on hover seems annoying, I think it would be better implementing it on click instead